sdk/samples/storage/blobs/src/blobs_client_example.c

```diff
@@ -17,13 +17,12 @@
  * 4) Get payload from response and parse it
  */
 
-#include <azure/core/az_context.h>
+#include <azure/az_core.h>
```

Comment:
Open question: Should these headers be in `/inc/azure/core/az_core.h` or is it better for them to be one-level higher, in `/inc/azure/az_core.h`.

**Either:**
/inc/azure/core/az_core.h
/inc/azure/iot/az_iot.h

**OR:**
/inc/azure/az_core.h
/inc/azure/az_iot.h

I prefer the latter.

Reply:
vote for latter